saleor/json_schemas/types/CheckoutCalculateTaxes.d.ts

```diff
@@ -0,0 +1,28 @@
+/* eslint-disable */
+/**
+ * This file was automatically generated by json-schema-to-typescript.
+ * DO NOT MODIFY IT BY HAND. Instead, modify the source JSONSchema file,
+ * and run json-schema-to-typescript to regenerate this file.
+ */
+
+export type ShippingTaxRate = number | string;
+export type ShippingPriceGrossAmount = number | string;
+export type ShippingPriceNetAmount = number | string;
+export type TaxRate = number | string;
+export type TotalGrossAmount = number | string;
+export type TotalNetAmount = number | string;
+export type Lines = LineCalculateTaxesSchema[];
+
+export interface CalculateTaxesSchema {
+  shipping_tax_rate: ShippingTaxRate;
+  shipping_price_gross_amount: ShippingPriceGrossAmount;
+  shipping_price_net_amount: ShippingPriceNetAmount;
+  lines?: Lines;
+  [k: string]: unknown;
```

Comment:
This is strange - and I think we should get rid of it if possible.

That means typescript will allow ANY field to be returned.

For example if I put `lins` it will accept that field
For required fields line `lines` there still be an error. But if we have

```
metadata?: Record // object or nothing
```

and then 

```
metdata: {} // with typo
```

typescript will accept that silently

I think `additionalProperties` field should be `false` (assuming you use https://www.npmjs.com/package/json-schema-to-typescript)